WearOAuth/oauth-pkce/src/main/java/com/example/android/wearable/oauth/pkce/AuthPKCEActivity.kt

```diff
@@ -35,22 +51,46 @@ class AuthPKCEActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_auth)
-        val viewModel by viewModels<AuthPKCEViewModel>()
-
-        // Start the OAuth flow when the user presses the button
-        findViewById<View>(R.id.authenticateButton).setOnClickListener {
-            viewModel.startAuthFlow()
-        }
-
-        // Show current status on the screen
-        viewModel.status.observe(this) { statusText ->
-            findViewById<TextView>(R.id.status_text_view).text = resources.getText(statusText)
-        }
+        setContent { PKCEApp(pkceViewModel = viewModel()) }
+    }
+}
 
-        // Show dynamic content on the screen
-        viewModel.result.observe(this) { resultText ->
-            findViewById<TextView>(R.id.result_text_view).text = resultText
+@OptIn(ExperimentalHorologistApi::class)
+@Composable
+fun PKCEApp(pkceViewModel: AuthPKCEViewModel) {
+    AppScaffold {
+        val uiState = pkceViewModel.uiState.collectAsState()
+        val localContext = LocalContext.current
+        val columnState = rememberResponsiveColumnState(
+            contentPadding = ScalingLazyColumnDefaults.padding(
+                first = ScalingLazyColumnDefaults.ItemType.Unspecified,
```

Comment:
with the content below, this should be first = Text & last = Text

Reply:
Done.